bin/compile

```diff
@@ -286,6 +286,11 @@ source "$BIN_DIR/steps/sqlite3"
 buildpack_sqlite3_install
 mtime "sqlite3.install.time" "${start}"
 
+# Source compile variables
+if [ -f .python-compile-env ]; then
```

Comment:
@tomasbielskis `-f` might be better as `-s`, so unless file exists and contains something, do nothing

Reply:
Changed it, it's reflected in #750